docs/README.md

```diff
@@ -13,6 +13,24 @@ Want to deploy the demo and see it in action? Start here.
 - [Docker](./docker_deployment.md)
 - [Kubernetes](./kubernetes_deployment.md)
 
+### Language Feature Reference
+
+Want to understand how a particular language's instrumentation works? Start
+here.
+
+| Language      | Auto Instrumentation                                                                                                | Manual Instrumentation                                                                                              |
+|---------------|---------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
+| Cpp           |                                                                                                                     |                                                                                                                     |
```

Comment:
I'd use C++ and .NET formats. However, we seem to have some variance in other tables.

Reply:
I also like C++ and .NET

Reply:
+1 `C++` and `.NET`